.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,313 @@
+name: Pull Request
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*.*.*'
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+  Test:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Dep
+      run: |
+        curl -L https://github.com/linuxsuren/http-downloader/releases/latest/download/hd-linux-amd64.tar.gz | tar xzv
```

Comment:
We should use `http-downloader` with version specified here instead of latest.

Reply:
Good point.